UPDATING.md

```diff
@@ -59,6 +59,26 @@ https://developers.google.com/style/inclusive-documentation
 
 -->
 
+### Cli use exactly single character for short option
+
+Use exactly one single character for short option in Airflow cli, New commands are available according to the following table:
+
+| Old command                                        | New command                                       |
+|----------------------------------------------------|---------------------------------------------------|
+| ``airflow (dags|tasks|scheduler) [-sd, --subdir]`` | ``airflow (dags|tasks|scheduler) [-S, --subdir]`` |
+| ``airflow tasks test [-dr, --dry_run]``            | ``airflow tasks test [-dr, --dry_run]``           |
+| ``airflow dags backfill [-dr, --dry_run]``         | ``airflow dags backfill [-n, --dry_run]``         |
+| ``airflow tasks clear [-dx, --dag_regex]``         | ``airflow tasks clear [-R, --dag_regex]``         |
+| ``airflow kerberos [-kt, --keytab]``               | ``airflow kerberos [-k, --keytab]``               |
+| ``airflow tasks run [-int, --interactive]``        | ``airflow tasks run [-N, --interactive]``         |
```

Comment:
How about `-I` ?

Reply:
We already have `-I, --ignore_depends_on_past` in `airflow tasks run`, so I use another letter

Reply:
There's already a section in updating about moving the commands around -- lets place this along-side that/under the same heading.

Reply:
Put in the same header? I thought we should make the laster change in the TOP before. So the correct way is combine the similar change?